modules/ROOT/pages/service-mesh-overview-and-landing-page.adoc

```diff
@@ -15,30 +15,33 @@ To alleviate issues arising from differences in these microservices, Anypoint Se
 
 == Business Solution
 
-Growing business demands for diverse deployment environments also introduces a need for rapidly deployable and manageable microservices architecture. In a microservice architecture, you build smaller, targeted services instead of large monolithic applications. These services are faster to build, easier to maintain, and help foster agility. 
+Growing business demands for diverse deployment environments introduces a need for rapidly deployable and manageable microservices architecture. In a microservice architecture, you build smaller, targeted services instead of large monolithic applications. These services are faster to build, easier to maintain, and help foster agility. 
 
-In a common enterprise setup, there are numerous teams building services that connect and expose functionalities across all sorts of systems: mainframes, SaaS apps, IoT devices, and so on. The result is usually a siloed environment, with teams managing these apps in different ways. 
+In a common enterprise setup, numerous teams are engaged in building services that connect and expose functionalities across all sorts of systems: mainframes, SaaS apps, IoT devices, and so on. The result is a siloed environment, with teams managing these applications in different ways. 
 
-With Anypoint Service Mesh from MuleSoft you can connect your applications, data, and devices through APIs in an applications network. You can manage, govern and secure your application network within Anypoint Platform.
+With Anypoint Service Mesh, you can connect your applications, data, and devices through APIs in an applications network. You can manage, govern, and secure your application network using Anypoint Platform.
```

Comment:
in an applications network > in an **application** network.

Reply:
Done! Thanks!